reviews-receiver/src/common/api-key.middleware.ts

```diff
@@ -0,0 +1,24 @@
+import { Request, Response, NextFunction } from 'express'
+import ApiKey from '../models/api-key.model'
+
+const checkApiKey = async (req: Request, res: Response, next: NextFunction) => {
+  const apiKey = req.header('x-api-key')
+
+  if (!apiKey) {
+    res.status(401).json({ message: 'API key is missing' })
+    return
```

Comment:
really doesn't matter but you can return the `res.status().json()` line instead of empty return
```suggestion
    return res.status(401).json({ message: 'API key is missing' })
```